tests/pr_test.go

```diff
@@ -78,3 +80,45 @@ func TestRunUpgradeBasicExample(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func setupOptionsOverride(t *testing.T, prefix string) *testhelper.TestOptions {
+
+	sshPublicKey := sshPublicKey(t)
+
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  overrideExampleTerraformDir,
+		Prefix:        prefix,
+		ResourceGroup: resourceGroup,
+		Region:        region,
+		TerraformVars: map[string]interface{}{
+			"ssh_key": sshPublicKey,
+		},
+	})
+
+	return options
+}
+
+func TestRunOverrideExample(t *testing.T) {
+	t.Parallel()
+
+	options := setupOptionsOverride(t, "slz-json")
+
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+// Re-introduce when initial version is in master
```

Comment:
The source code of the module is already in master, so you could enable this now. however because there are static names used in the override.json, I am wondering if the two tests would start to clash if you did enable this? Also if two PRs were opened around the same time, they would also clash as well. Not sure what best way to handle it is, unless we some how dynamically generated the override json for every test run?

Reply:
The example that is added in this PR is not in main yet. Once this is merged, the upgrade test for this example can be enabled. The prefix should be used so there should not be clashes.

Reply:
ah yes right, the example needs to be in main too duh. 
Is prefix still used when using override.json though? For example, I see prefix defined in the override.json in places. Also won't things like these clash?
```
"resource_group": "slz-service-rg"
"collector_bucket_name": "atracker-bucket"
```

Reply:
@ocofaigh - The prefix is added to the values in the json file. For instance, when running the tests we end up with "slz-json-o5duxj-slz-service-rg" based on the logic at https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/add-custom-example/tests/pr_test.go#L105